src/access_nri_intake/source/builders.py

```diff
@@ -590,42 +527,18 @@ def parser(cls, file):
             realm = match_groups[1]
 
             realm_mapping = {"atm": "atmos", "ocn": "ocean", "ice": "seaIce"}
-            realm = realm_mapping[realm]
-
-            (
-                filename,
-                file_id,
-                _,
-                frequency,
-                start_date,
-                end_date,
-                variable_list,
-                variable_long_name_list,
-                variable_standard_name_list,
-                variable_cell_methods_list,
-                variable_units_list,
-            ) = cls.parse_access_ncfile(file)
+
+            nc_info = cls.parse_access_ncfile(file)
+            ncinfo_dict = nc_info.to_dict()
 
             # Remove exp_id from file id so that members can be part of the same dataset
-            file_id = re.sub(exp_id, "", file_id).strip("_")
-
-            info = {
-                "path": str(file),
-                "realm": realm,
-                "variable": variable_list,
-                "frequency": frequency,
-                "start_date": start_date,
-                "end_date": end_date,
-                "member": exp_id,
-                "variable_long_name": variable_long_name_list,
-                "variable_standard_name": variable_standard_name_list,
-                "variable_cell_methods": variable_cell_methods_list,
-                "variable_units": variable_units_list,
-                "filename": filename,
-                "file_id": file_id,
-            }
-
-            return info
+            ncinfo_dict["file_id"] = re.sub(exp_id, "", ncinfo_dict["file_id"]).strip(
+                "_"
+            )
```

Comment:
Not critical, but I think this line-breaking/indentation looks a bit better:

```suggestion
            ncinfo_dict["file_id"] = re.sub(
                exp_id, "", ncinfo_dict["file_id"]
                ).strip("_")
```

Reply:
I agree - the formatter results here are definitely difficult to read. How does this look to you?
```suggestion
ncinfo_dict["file_id"] = re.sub(
                exp_id,
                "",
                ncinfo_dict["file_id"],
            ).strip("_")
```